File: platform/sdk/python/examples/cloud/cloud-013_sts_assume_3600s_reject.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
result = client.stsAssume("arn:mox:iam::role/long", 3600)
if result["ok"]:
    sys.exit(1)
print("XJ-OK: cloud-013_sts_assume_3600s_reject")
sys.exit(0)

File: platform/sdk/nodejs/examples/cloud/cloud-013_sts_assume_3600s_reject.js
const { CloudClient } = require("../../mox-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
const result = client.stsAssume("arn:mox:iam::role/long", 3600);
if (result.ok) process.exit(1);
console.log("XJ-OK: cloud-013_sts_assume_3600s_reject");
process.exit(0);
